### PR TITLE
PLAT-1040: Update notification styles

### PR DESCRIPTION
**Pull request by mihaelpalfi**
[JIRA ticket](https://reciprocitylabs.atlassian.net/browse/PLAT-1040)

#### Description
As per JIRA ticket

image

#### ChangeLOG

- [ ] I have added all notable deployment/development environment (onprem!) changes to [CHANGELOG.md](https://github.com/reciprocity/zen-ui/blob/main/README.md)

#### Useful links
[Zen UI readme](https://github.com/reciprocity/zen-ui/blob/main/README.md)

[PR review guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/pull_request_reviews.md)

[Git commit guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/commit_guidelines.md)

### CODE REVIEW

File: src/components/zen-notification/zen-notification.tsx [hunk 1]
```diff
@@ -68,14 +73,10 @@ export class ZenNotification {
...
           </div>
           <div class="row">
-            <div class="icon-container">
-              <div class={icon}>{getIcon(this.variant)}</div>
-            </div>
-            <div class="content-container">
-              <div class="title">{this.heading}</div>
-              <div class="message">{this.message}</div>
-            </div>
+            <div class={icon}>{getIcon(this.variant)}</div>
```
Comment (f-jauregui) [inline src/components/zen-notification/zen-notification.tsx:76]: You can use `<zen-icon>` now to achieve this.
In line 73 can be also changed to use `<zen-icon>`.

**Review by f-jauregui — APPROVED** [review]
I would just change the Icon usage. Otherwise, fantastic job Mihael. Those notifications look great!

**Review by mplesec — APPROVED** [review]
Awesome 👍

Only one nitpick, these should be vertically aligned as well:
Screenshot 2021-02-10 at 15 15 08

File: src/components/zen-notification/functions.ts [hunk 2]
```diff
@@ -1,21 +1,21 @@
 import { renderIcon } from '../helpers/fa-icons';
-import { faBell, faCheck, faExclamation, faTimes } from '@fortawesome/pro-solid-svg-icons';
+import { faInfoCircle, faCheckCircle, faExclamationTriangle, faTimesHexagon } from '@fortawesome/pro-light-svg-icons';
```
Comment (chavlji) [inline src/components/zen-notification/functions.ts:2]: Please rename this file to `helpers.ts`

File: src/components/zen-notification/zen-notification.tsx (hunk 1 above)
Comment (chavlji) [reply src/components/zen-notification/zen-notification.tsx:76]: True 👍

**Review by chavlji — CHANGES_REQUESTED** [review]
Nice! Though styling seems a bit off according to designs (designs are on right). Use [perfect pixel](https://chrome.google.com/webstore/detail/perfectpixel-by-welldonec/dkaagdgjmgdmbnecmcefdhjekcoceebi?hl=en) to match it 100%.

Screenshot 2021-02-11 at 09 22 08

Screenshot 2021-02-11 at 09 21 53

And one more thing: please remove prop `message` and replace it with the default slot. It's way more natural.

File: src/components/zen-notification/zen-notification.tsx (hunk 1 above)
Comment (mihaelpalfi) [reply src/components/zen-notification/zen-notification.tsx:76]: Done

File: src/components/zen-notification/functions.ts (hunk 2 above)
Comment (mihaelpalfi) [reply src/components/zen-notification/functions.ts:2]: Done

**Review by chavlji — APPROVED** [review]

**Pull request merged**